### DISCUSSION TITLE
How to use multiple  properties in Popup

### DISCUSSION THREAD

**Discussion by creinemann** [bozdoz/wp-plugin-leaflet-map#153] [Q&A]
I am trying to display all of the properties in this geojson file. 
`{
   "type": "FeatureCollection",
   "features": [
  {
    "type": "Feature",
    "geometry": {
       "type": "Point",
       "coordinates":  [ -88.800377,43.00694 ]
    },
    "properties": {
    "Station_name":"USRadioguy",
    "Satellites_received":"GOES 16,17, Himawari8",
    "Province_State":"Wisconsin",
    "Country":"USA",
    "URL":"https://Usradioguy.com"
    }
  },
  {
    "type": "Feature",
    "geometry": {
       "type": "Point",
       "coordinates":  [ -80.1918,25.7617 ]
    },
    "properties": {
    "Station_name":"Teds",
    "Satellites_received":"NOAA",
    "Province_State":"Florida",
    "Country":"USA",
    "URL":""
    }
  },`
  
  Using this shortcode
  `[leaflet-map doubleClickZoom=true scrollwheel=true]
[leaflet-geojson src="https://usradioguy.com/map/ground.json" popup_property="Station_name"]`

I have tried adding `popup_property="Station_name; Satellites_received; Province_State; Country; URL"` like this to no avail, how could I show all the properties and a link with the URL?

**Comment by bozdoz** [comment]
Look up 'table-view' on the main README.md
